doc/changelog/13-extraction/19906-add-file-pragma.rst

```diff
@@ -0,0 +1,2 @@
+- **Added:**
+  The :flag:`Extraction File Pragma` (unset by default) inserts compiler pragmas in the beginnig of a generated file (`https://github.com/coq/coq/pull/19906`_, by Daniil Iaitskov).
```

Comment:
```suggestion
  The :opt:`Extraction File Pragma` (unset by default) inserts a compiler pragma at the beginning of a generated file (`#19906 <https://github.com/coq/coq/pull/19906>`_, by Daniil Iaitskov).
```

Your link was misformatted, causing the doc build to fail.
Also using flag instead of opt will cause the build to fail.
I think it should be "a pragma" because IINM the option only supports one pragma at a time.

Reply:
FWIW: your Github description has "Daneel" rather than "Daniil" used here.

Reply:
Thanks for the explanation. I updated my name in GitHub account.